snippets/zig.json

```diff
@@ -1,29 +1,25 @@
 {
-	"Import": {
-		"prefix": "import",
-		"body": ["const ${1} = @import(\"${1}\")"],
-		"description": "Importing Librarys"
-	},
-	"CImport": {
-		"prefix": "cimport",
-		"body": [
-			"const c = @cImport({",
-				"@cDefine(\"${1}\")",
-				"});"
-		],
-		"description": "Importing C Header Files"
-	},
-	"buildExe": {
-		"prefix": "bExe",
-		"body": [
-			"const exe = b.addExecutable(.{",
-				".name = \"${1}",\",
-				".root_source_file = b.path(\"${2: path}\"),",
-				".target = target,",
-				".optimize = optimize,",
-				"});",
-				"\n\n\n b.installArtifact(exe);",
-		],
-		"description": "Building an exe"
-	}
+  "Import": {
+    "prefix": "import",
+    "body": ["const ${1} = @import(\"${1}\")"],
+    "description": "Importing Librarys"
```

Comment:
@max2000warlord there is a typo here, should be spelled "libraries"

Reply:
Didn't notice that lol. Looks like it's in the original code, too. Fixed.